charts/connection-manager/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: "1.0.0"
+appVersion: "1.1.0"
 description: Connection Manager Helm chart for Kubernetes
 name: connection-manager
-version: 1.0.0
+version: 1.1.0
```

Comment:
Should we always changes both `appVersion` and `version`?  and is _yes_, should they always have the same value?

Reply:
The current case adds a new feature in both helm and the app, that's why they match.